dsymprocessor/config.go

```diff
@@ -0,0 +1,67 @@
+package dsymprocessor
+
+import "time"
+
+// Config defines configuration for the symbolicator processor.
+type Config struct {
+	// SymbolicatorFailureAttributeKey is the attribute key that will be set to
+	// true if the symbolicator fails to fully symbolicate a stack trace.
+	SymbolicatorFailureAttributeKey string `mapstructure:"symbolicator_failure_attribute_key"`
+
+	// MetricKitStackTraceAttributeKey is the attribute key that contains the metrickit
+	// stack trace.
+	MetricKitStackTraceAttributeKey string `mapstructure:"metrickit_stack_trace_attribute_key"`
+
+	// OutputMetricKitStackTraceAttributeKey is the attribute key that contains the
+	// symbolicated metrickit stack trace.
+	OutputMetricKitStackTraceAttributeKey string `mapstructure:"output_metrickit_stack_trace_attribute_key"`
+
+	// preserveStackTrace is a config option that determines whether to keep the
+	// original stack trace in the output.
+	// TODO: use this
+	PreserveStackTrace bool `mapstructure:"preserve_stack_trace"`
```

Comment:
@martin308 I swiped this config from the source maps config but looking at this processor: we grab the stack from `metrickit.diagnostic.crash.exception.stacktrace_json` (or whatever is given in the config `metrickit_stack_trace_attribute_key`) and then we store it in `metrickit.diagnostic.crash.exception.stacktrace` (or whatever is given in the config `output_metrickit_stack_trace_attribute_key`). We're not overwriting the original error info like the source map processor does. 

Do we think we should support this option and _delete_ the contents of the `metrickit.diagnostic.crash.exception.stacktrace_json` field if it's `false`?

Reply:
also followup question, I guess: should we be storing the symbolicated error in a different attribute? ie. an otel standard attribute?

Reply:
Yeh I think you're right that removing the `stacktrace_json` key if it's false is probably the more consistent way of doing it. 

@pkanal may have more of a sense for the attribute that we store the symbolicated trace in. I think that we will have to use the semantic conventions to light up the errors UI bits

Reply:
Yes, we'll want `exception.stacktrace` for the symbolicated stack trace [according to OTel semconv](https://opentelemetry.io/docs/specs/semconv/registry/attributes/exception/#exception-attributes).

Reply:
ok i've updated the [default config](https://github.com/honeycombio/opentelemetry-collector-symbolicator/pull/53/files#diff-412ce9e8c76fbf487b3a8c92a7f57747d8433e05f10ca8b420f9748e74b9a8f7R22)